interface-definitions/include/haproxy/tcp-request.xml.i

```diff
@@ -0,0 +1,18 @@
+<!-- include start from haproxy/tcp-request.xml.i -->
+<node name="tcp-request">
+  <properties>
+    <help>TCP request directive</help>
+  </properties>
+  <children>
+    <leafNode name="inspect-delay">
+      <properties>
+        <help>Set the maximum allowed time to wait for data during content inspection</help>
+        <valueHelp>
+          <format>u32:1-65535</format>
+          <description>The timeout value specified in milliseconds</description>
```

Comment:
You should use “constraint”
For example https://github.com/vyos/vyos-1x/blob/4e64267a408062f18140c2a1c6eb38f023cd4c76/interface-definitions/container.xml.in#L200

Use “commit —amend” and “git push —force” to overwrite the previous commit.

Reply:
done